tests/testthat/test-autosyn.R

```diff
@@ -88,13 +88,15 @@ test_that("flywire_neurons_add_synapses works", {
   skip_if(inherits(token, "try-error"),"Skipping live flywire tests")
   skip_if_not(reticulate::py_module_available("cloudvolume"),
               "Skipping live flywire tests requiring python cloudvolume module")
-  expect_is(neuron <- readRDS("testdata/flywire_neuron_skeleton.rds"), 'neuronlist')
+  expect_is(fwskel <- readRDS(testthat::test_path("testdata/flywire_neuron_skeleton.rds")), 'neuronlist')
   skip_if(is.null(ntpredictions_tbl()), "Skipping tests relying on sqlite databases")
   if(!is.null(ntpredictions_tbl())) {
-    expect_is(neuron.syn = flywire_neurons_add_synapses(x=neuron, transmitters = TRUE, method = "auto"), c("neuronlist"))
-    expect_is(neuron.syn[[1]]$transmitter.predictions,'table')
+    expect_is(neuron.syn <- flywire_neurons_add_synapses(x=fwskel, transmitters = TRUE, method = "auto"), c("neuronlist"))
```

Comment:
@alexanderbates please note the the fix here. Great that you added the test, but even better if you run them! (you can't have since there was a syntax error ...)

Reply:
noted. srry, i pushed before running off for the day. I suppose I have a bad habit of pushing WIP, which I should curb

Reply:
It would be best if you run all tests before pushing. I sometimes don't run check if I am on a branch (where travis will run check for me). At a minimum I run the test file that should be most affected. e.g. using the Run Tests button for `test-autosyn.R`
![image](https://user-images.githubusercontent.com/23763/105648139-b9824500-5ea1-11eb-8e6e-e578fcda4525.png)

Reply:
So the issue was admittedly sloppiness + the fact that at that time I still had not updated my python cloud-volume module. Which I have now done.